sql-cli/sql_cli/configuration.py

```diff
@@ -68,6 +69,7 @@ def from_yaml_to_config(self) -> Config:
             environment=self.environment,
             airflow_home=global_yaml_config.get("airflow", {}).get("home"),
             airflow_dags_folder=global_yaml_config.get("airflow", {}).get("dags_folder"),
+            data_dir=global_yaml_config.get("general", {}).get("data_dir"),
```

Comment:
Do we want the `data_dir` to be environment specific or global? It may simplify to have it global, as a starting point

Reply:
I think global is fine to start. We can make it environmental later if that matches user requests/usage patterns.